packages/client/src/store/actions/organizationOperations.ts

```diff
@@ -37,6 +46,7 @@ export const updateOrganization =
       );
       setSubmitting(false);
     } catch (error) {
+      console.log({ error });
```

Comment:
If we're doing this, it should be `console.error`. It's always disputable whether or not we should log to console in production, but this way the sentry will pick it up as an error (I think), so we get an added benefit there.